### DISCUSSION TITLE
Workflow contextId is getting lost on suspend

### DISCUSSION THREAD

**Discussion by amanshrm365** [elsa-workflows/elsa-core#1533] [Q&A]
Hi i am trying with the start at activity whenever i put a activity in suspend state and when it comes back onResumeAsync method it's workflow context become null even contextId get's lost.

![image](https://user-images.githubusercontent.com/42913145/133543078-17589184-3f56-47ae-b714-25c7a6abe414.png)
Whenever first activity goes into suspend state the next activity lost it contextId.

is there any way to persist the context Id when suspend is occured?

**Comment by sfmskywalker** [comment]
What package version are you using? There was a problem with context not being loaded in time, but I’ve not seen the context ID getting lost.

If you are seeing this behavior with the latest prerelease (2.3 from MyGet) then I would love a repro that I can investigate.
